src/TLE.ts

```diff
@@ -1280,61 +1273,61 @@ export class Token {
     }
 
     public static createLeftParenthesis(startIndex: number): Token {
-        return Token.create(TokenType.LeftParenthesis, startIndex, "(");
+        return new Token(TokenType.LeftParenthesis, startIndex, "(");
     }
 
     public static createRightParenthesis(startIndex: number): Token {
-        return Token.create(TokenType.RightParenthesis, startIndex, ")");
+        return new Token(TokenType.RightParenthesis, startIndex, ")");
     }
 
     public static createLeftSquareBracket(startIndex: number): Token {
-        return Token.create(TokenType.LeftSquareBracket, startIndex, "[");
+        return new Token(TokenType.LeftSquareBracket, startIndex, "[");
     }
 
     public static createRightSquareBracket(startIndex: number): Token {
-        return Token.create(TokenType.RightSquareBracket, startIndex, "]");
+        return new Token(TokenType.RightSquareBracket, startIndex, "]");
     }
 
     public static createComma(startIndex: number): Token {
-        return Token.create(TokenType.Comma, startIndex, ",");
+        return new Token(TokenType.Comma, startIndex, ",");
     }
 
     public static createPeriod(startIndex: number): Token {
-        return Token.create(TokenType.Period, startIndex, ".");
+        return new Token(TokenType.Period, startIndex, ".");
     }
 
     public static createWhitespace(startIndex: number, stringValue: string): Token {
```

Comment:
>createWhitespace [](start = 18, length = 16)

Nit: unrelated to this PR, seems like this should've asserted that stringValue is actually whitespace, since other createXXX functions check the value